src/referentie_tabellen/admin.py

```diff
@@ -7,6 +9,39 @@
     Unit,
 )
 
+admin.site.unregister(User)
+
+
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
+    """Custum UsterAdmin"""
+
+    def get_form(self, request, obj=None, **kwargs):
+        form = super().get_form(request, obj, **kwargs)
+        is_superuser = request.user.is_superuser
+        disabled_fields = set()
+
+        if not is_superuser:
+            disabled_fields |= {
+                "is_superuser",
+                "user_permissions",
+            }
+
+        # Prevent non-superusers from editing their own permissions
+        if not is_superuser and obj is not None and obj == request.user:
```

Comment:
Deze if kan je verkorten door `obj is not None` weg te halen. Je test al of obj gelijk staat aan request.user, dus dan is hij sowieso niet None.